File: 프로그래머스/연습문제_2016년/2016년.py
def solution(a, b):
    answer = ''
    month = [31, 29, 31,30, 31, 30, 31, 31, 30, 31, 30, 31]
    day = ['FRI', 'SAT', 'SUN', 'MON', 'TUE','WED', 'THU']
    sum = 0
    for i in range(a-1):
        sum+=month[i]
    sum+=b
    answer = day[sum%7 -1]
    return answer

File: 프로그래머스/연습문제_2016년/2016년.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

string solution(int a, int b) {
    string answer = "";
    int value = 0;
    vector<int>month{31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    vector<string>date {"FRI", "SAT","SUN", "MON", "TUE", "WED", "THU"};
    for(int i=0; i <a-1; i++){
        value = (value + month[i]% 7) % 7;
    }
    value =(value+ b % 7 -1)%7;

    answer = date[value];
    return answer;
}